week1/homework/src/server.js

```diff
@@ -10,6 +10,53 @@ function createServer(port) {
 
   const server = http.createServer((request, response) => {
     // TODO: Write your homework code here
+
+    switch (request.url) {
+      case '/add':
+        response.writeHead(200, {
+          'Content-Type': 'application/json'
+        });
+
+        response.end(JSON.stringify({
+          state: ++state
+        }, null, 2));
```

Comment:
From all the `case`s you have,
you have some extras no need for them unless you are using it for let's say writing to a file or view the object somewhere you want it to be beautified.

```suggestion
        }));
```

Reply:
I have fixed it in writeResponse function.